package.json

```
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "start": "parcel -d .cache/dev src/index.html",
+    "start": "parcel -d .cache src/index.html",
```

hm, even that didn't help. In the end I went with 
```suggestion
    "start": "rm -rf .cache && parcel src/index.html",
```

---

The reason I used `.cache/dev` is that if I have a dev server started in a tab and I simultaneously try to build in another tab it used to mess up the output. Not sure what the case is right now.

You might want to give Parcel 2 a try which recently went into Beta. I use it in other projects (I'm not using Elm anymore) and it works great, it's faster and produces smaller JS output. It also does not need `-d .cache`. Check out the parcel related tasks at https://github.com/utkarshkukreti/reaml-opinionated-starter/blob/master/package.json and the Tailwind config files in the same repo. It also uses the latest tailwind and new builtin css purging feature.

---

> (I'm not using Elm anymore

---

Thanks, I'll check it out.

> I'm not using Elm anymore

I'm curios - why not?

---

> I'm curios - why not?

I found Elm to be a lot less powerful than what other languages like OCaml/ReasonML and PureScript and also the Elm Architecture to be very verbose compared to what React and others offer.

I wrote and use https://github.com/utkarshkukreti/reaml now which is React + OCaml/ReasonML.

I still consider Elm to be the best way to start learning functional programming for frontend.

---

:ok_hand: